### PR TITLE
Bump the runtime to 3.34

### PR DESCRIPTION
**Pull request by Kekun**

### CODE REVIEW

**Comment by alatiera** [comment]
About the build failure, I plan on doing a new release in the weekend.

About the runtime bump, will get on it along with the release but it will be 3.32 most likely due to a gstreamer bug, unless that's fixed (somewhat likely) till the weekend.

**Comment by Kekun** [comment]
@alatiera 3.32 is fine, I am just trying to reduce the number of GNOME runtimes I have installed from 5 to 3 (I can handle master, 3.34 and 3.32, but 3.30 and 3.28 should go away).

**Comment by TingPing** [comment]
Remove and migrate DConf permissions.

**Comment by go-ten** [comment]
Hi,
Is there any news on this request?
Thanks,

**Comment by alatiera** [comment]
bot, build

File: org.gnome.Podcasts.json [hunk 1]
```diff
@@ -8,17 +8,11 @@
     ],
     "command" : "gnome-podcasts",
     "finish-args" : [
-        "--filesystem=xdg-run/dconf",
```
Comment (TingPing) [inline org.gnome.Podcasts.json:11]: Add the migration metadata. `"--metadata=X-DConf=migrate-path=/org/gnome/whatever/",`

File: org.gnome.Podcasts.json [hunk 2]
```diff
@@ -8,17 +8,11 @@
...
-        "--talk-name=ca.desrt.dconf",
-        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
         "--share=network",
         "--share=ipc",
-        "--socket=x11",
+        "--socket=fallback-x11",
         "--socket=wayland",
-        "--socket=pulseaudio",
-        "--device=dri",
-        "--own-name=org.mpris.MediaPlayer2.Podcasts"
```
Comment (TingPing) [inline org.gnome.Podcasts.json:21]: Just confirming upstream renamed this?

File: org.gnome.Podcasts.json [hunk 3]
```diff
@@ -8,17 +8,11 @@
...
-        "--filesystem=~/.config/dconf:ro",
-        "--talk-name=ca.desrt.dconf",
-        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
         "--share=network",
         "--share=ipc",
-        "--socket=x11",
+        "--socket=fallback-x11",
         "--socket=wayland",
-        "--socket=pulseaudio",
-        "--device=dri",
```
Comment (TingPing) [inline org.gnome.Podcasts.json:20]: Why was this removed?

Comment (alatiera) [reply org.gnome.Podcasts.json:20]: Was planned for accessing gl with the `gstglsink` but plans changed and it won't be needed.

File: org.gnome.Podcasts.json (hunk 2 above)
Comment (alatiera) [reply org.gnome.Podcasts.json:21]: Yeap, here's the upstream [MR](https://gitlab.gnome.org/World/podcasts/merge_requests/124)

Even mentioned your blog post :P (on irc)

File: org.gnome.Podcasts.json (hunk 1 above)
Comment (alatiera) [reply org.gnome.Podcasts.json:11]: Good catch, thanks!

**Comment by alatiera** [comment]
bot, build

(Previous one failed due to a 503)

**Review by TingPing — APPROVED** [review]
Looks good.
